### PR TITLE
Added corrections to android getting started guide

### PR DESCRIPTION
**Pull request by timMikkelsen**

### CODE REVIEW

File: src/android/v3/getting-started/directions.md [hunk 1]
```diff
@@ -194,6 +202,33 @@ Now we will implement logic to our `NavigationFragment` that we can put into our
 
 Here we'll use a `viewpager` to allow the user to switch between each step, as well as display a "close" button so we are able to remove the route and the bottom sheet from the activity.
 
+We will start by making a getter for our `MPDirectionsRenderer` the we store on `MapsActivity`
```
Comment (marksdk) [inline src/android/v3/getting-started/directions.md:205]:
```suggestion
We will start by making a getter for our `MPDirectionsRenderer` that we store on `MapsActivity`:
```

File: src/android/v3/getting-started/map.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ eleventyNavigation:
 
 ### Initialize MapsIndoors
 
-Place the following initialization code in the `onCreate` method in the `MapsActivity` that displays the Google map. You should also assign the `mapFragment` view to a local variable as we will use this later to initialize `MapControl`:
+We start by initializing `MapsIndoors`. `MapsIndoors` is used to get all and store all references to MapsIndoors specific data. This includes access to all MapsIndoors specific Geodata.
```
Comment (marksdk) [inline src/android/v3/getting-started/map.md:19]:
```suggestion
We start by initializing `MapsIndoors`. `MapsIndoors` is used to get all and store all references to MapsIndoors-specific data. This includes access to all MapsIndoors-specific geodata.
```

File: src/android/v3/getting-started/map.md [hunk 3]
```diff
@@ -16,7 +16,9 @@ eleventyNavigation:
...
+
+Place the following initialization code in the `onCreate` method in the `MapsActivity` that displays the Google map. You should also assign the `mapFragment` view to a local variable as we will use this later to initialize `MapControl` inside the onCreate after it has been created:
```
Comment (marksdk) [inline src/android/v3/getting-started/map.md:21]:
```suggestion
Place the following initialization code in the `onCreate` method in the `MapsActivity` that displays the Google map. You should also assign the `mapFragment` view to a local variable, as we will use this later to initialize `MapControl` inside the onCreate, after it has been created:
```

File: src/android/v3/getting-started/map.md [hunk 4]
```diff
@@ -48,7 +58,13 @@ mapFragment.view?.let {
 
 If you are not a customer you can use this demo MapsIndoors API key `d876ff0e60bb430b8fabb145`.
 
-In your `onMapReady` callback function, use the `MapControl` class to set up a Google map with MapsIndoors Venues, Buildings and Locations:
+### Initialize MapsControl
+
+We now want to add all the data we get by initializing `MapsIndoors` to our Google Map. This is done by Initialzing MapControl onto the Google Map. MapControl is used as a layer between Google Maps and MapsIndoors.
```
Comment (marksdk) [inline src/android/v3/getting-started/map.md:63]:
```suggestion
We now want to add all the data we get by initializing `MapsIndoors` to our Google map. This is done by initializing `MapControl` onto the Google map. `MapControl` is used as a layer between Google Maps and MapsIndoors.
```

File: src/android/v3/getting-started/map.md [hunk 5]
```diff
@@ -48,7 +58,13 @@ mapFragment.view?.let {
...
+
+Here we use Google Map logic to apply Geodata onto the map. This also means we append logic onto a lot of the Google Maps listeners, which means that using google maps listeners directly might break intended behaviour of the MapsIndoors experience. We recommend to check our reference docs, and see if you can add a specific Listener through the MapControl and always use thoose when possible.
```
Comment (marksdk) [inline src/android/v3/getting-started/map.md:65]:
```suggestion
Here we use Google Maps logic to apply geodata onto the map. This also means we append logic onto many Google Maps listeners, which means that using Google Maps listeners directly might break intended behavior of the MapsIndoors experience. We recommend to check our reference docs, and see if you can add a specific `Listener` through the `MapControl` and always use those when possible.
```

File: src/android/v3/getting-started/map.md [hunk 6]
```diff
@@ -48,7 +58,13 @@ mapFragment.view?.let {
...
+
+Start by creating a `initMapControl` method we will use to initiate the MapControl and assign it to our Google Map:
```
Comment (marksdk) [inline src/android/v3/getting-started/map.md:67]:
```suggestion
Start by creating an `initMapControl` method which is used to initiate the `MapControl` and assign it to our Google map:
```

File: src/android/v3/getting-started/map.md [hunk 7]
```diff
@@ -122,6 +121,42 @@ private fun initMapControl(view: View) {
 </mi-tab-panel>
 </mi-tabs>
 
+In your `onMapReady` callback function, assign the `mMap` variable with the GoogleMap you get from the callback and call the initMapControl method with the `mMapView` you assigned in the `onCreate` to set up a Google map with MapsIndoors Venues, Buildings and Locations:
```
Comment (marksdk) [inline src/android/v3/getting-started/map.md:124]:
```suggestion
In your `onMapReady` callback function, assign the `mMap` variable with the `GoogleMap` you get from the callback and call the `initMapControl` method with the `mMapView` you assigned in the `onCreate` to set up a Google map with MapsIndoors Venues, Buildings and Locations:
```

File: src/android/v3/getting-started/new-project.md [hunk 8]
```diff
@@ -16,24 +16,34 @@ eleventyNavigation:
 
 This guide explains how to start using a MapsIndoors map in your Android application using the MapsIndoors Android SDK v3.
 
+We recommend using Android Studio for using this tutorial. Read how to set it up here: [Installing Android Studio](https://developer.android.com/studio/install)
+
+If you do not have a Android device, you can setup an emulator through Android studio, you can see how to do this here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```
Comment (marksdk) [inline src/android/v3/getting-started/new-project.md:21]:
```suggestion
If you do not have a Android device, you can setup an emulator through Android studio. You can see how to do that here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```

File: src/android/v3/getting-started/new-project.md [hunk 9]
```diff
@@ -16,24 +16,34 @@ eleventyNavigation:
...
+
+If you already have a Android device make sure to enable developer mode and USB debugging. Read about this here [Enable USB debugging](https://developer.android.com/studio/debug/dev-options#enable)
```
Comment (marksdk) [inline src/android/v3/getting-started/new-project.md:23]:
```suggestion
If you already have an Android device, make sure to enable developer mode and USB debugging. Read about this here: [Enable USB debugging](https://developer.android.com/studio/debug/dev-options#enable)
```

File: src/android/v3/getting-started/new-project.md [hunk 10]
```diff
@@ -16,24 +16,34 @@ eleventyNavigation:
...
 * Google Maps Android API
 
-You can get started in two ways, either by reviewing and modifying the [basic example](#basic-example) or do the [clean setup](#setup-mapsindoors).
+You can get started in two ways, either by reviewing and modifying the [basic example](#basic-example) or do the [clean setup](#setup-mapsindoors). We recommend following the basic example.
 
 ## Basic Example
 
 The tutorial will be based on you starting from our basic map implementation. This contains basic UI implementations together with layout files and drawables used to create the UI. You will then be guided through how to implement the MapsIndoors SDK into this app.
 
+The basic example contains a single `activity` app with already made `fragments` to host the different logic to get a complete app interracting with a map and `MapsIndoors` data.
```
Comment (marksdk) [inline src/android/v3/getting-started/new-project.md:36]:
```suggestion
The basic example contains a single `activity` app with already made `fragments` to host the different logic to get a complete app interacting with a map and `MapsIndoors` data.
```

File: src/android/v3/getting-started/new-project.md [hunk 11]
```diff
@@ -64,12 +74,16 @@ android {
 
 Add the following dependencies and the MapsIndoors maven repository:
 
+`Gson` and `okhttp` is used by MapsIndoors to function properly with network calls and deserializing.
+
+`play-services-maps` is used for the Google maps which MapsIndoors is build ontop of on Android.
```
Comment (marksdk) [inline src/android/v3/getting-started/new-project.md:79]:
```suggestion
`play-services-maps` is used for Google Maps which MapsIndoors is build on top of on Android.
```

File: src/android/v3/getting-started/search.md [hunk 12]
```diff
@@ -68,7 +68,7 @@ private fun search(searchQuery: String) {
 
 To be able to search we will use a text input field where a user can write what they want to search for. This is placed at the top of the MapsActivity
 
-To call our search method with the text in the search input field, we then add an `EditorActionListener` and a `OnClickListener` to the text input field and the search button in the `onCreate` of `MapsActivity`. Find the full `onCreate` example here: [MapsActivity.java](https://github.com/MapsIndoors/MapsIndoors-Getting-Started-Android/blob/master/app/src/main/java/com/example/mapsindoorsgettingstarted/MapsActivity.java#L53-L117) or [MapsActivity.kt](https://github.com/MapsIndoors/MapsIndoors-Getting-started-android-Kotlin/blob/main/app/src/main/java/com/example/mapsindoorsgettingstartedkotlin/MapsActivity.kt#L42-L106)
+To call our search method with the text in the search input field, we then add an `EditorActionListener` and a `OnClickListener` to the text input field and the search button in the `onCreate` of `MapsActivity`.
```
Comment (marksdk) [inline src/android/v3/getting-started/search.md:71]:
```suggestion
To call our search method with the text in the search input field, we add an `EditorActionListener` and a `OnClickListener` to the text input field, as well as the search button in the `onCreate` of `MapsActivity`.
```

File: src/android/v3/getting-started/directions.md (hunk 1 above)
Comment (marksdk) [inline src/android/v3/getting-started/directions.md:205]:
```suggestion
We will start by making a getter for our `MPDirectionsRenderer` that we store on `MapsActivity`:
```

**Review by marksdk — CHANGES_REQUESTED** [review]

File: src/android/v3/getting-started/new-project.md (hunk 8 above)
Comment (WJesper) [reply src/android/v3/getting-started/new-project.md:21]:
```suggestion
If you do not have an Android device, you can setup an emulator through Android studio, you can see how to do this here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```

**Review by WJesper — APPROVED** [review]

File: src/android/v3/getting-started/map.md [hunk 13]
```diff
@@ -16,7 +16,9 @@ eleventyNavigation:
 
 ### Initialize MapsIndoors
 
-Place the following initialization code in the `onCreate` method in the `MapsActivity` that displays the Google map. You should also assign the `mapFragment` view to a local variable as we will use this later to initialize `MapControl`:
+We start by initializing `MapsIndoors`. `MapsIndoors` is used to get all and store all references to MapsIndoors-specific data. This includes access to all MapsIndoors-specific geodata.
```
Comment (markhougaard) [inline src/android/v3/getting-started/map.md:19]:
```suggestion
We start by initializing `MapsIndoors`. `MapsIndoors` is used to get and store all references to MapsIndoors-specific data. This includes access to all MapsIndoors-specific geodata.
```

File: src/android/v3/getting-started/map.md [hunk 14]
```diff
@@ -16,7 +16,9 @@ eleventyNavigation:
...
+
+Place the following initialization code in the `onCreate` method in the `MapsActivity` that displays the Google map. You should also assign the `mapFragment` view to a local variable, as we will use this later to initialize `MapControl` inside the onCreate, after it has been created:
```
Comment (markhougaard) [inline src/android/v3/getting-started/map.md:21]:
```suggestion
Place the following initialization code in the `onCreate` method in the `MapsActivity` that displays the Google map. You should also assign the `mapFragment` view to a local variable, as we will use this later to initialize `MapControl` inside the `onCreate`, after it has been created:
```

File: src/android/v3/getting-started/new-project.md [hunk 15]
```diff
@@ -16,24 +16,34 @@ eleventyNavigation:
 
 This guide explains how to start using a MapsIndoors map in your Android application using the MapsIndoors Android SDK v3.
 
+We recommend using Android Studio for using this tutorial. Read how to set it up here: [Installing Android Studio](https://developer.android.com/studio/install)
+
+If you do not have a Android device, you can setup an emulator through Android studio. You can see how to do that here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```
Comment (markhougaard) [inline src/android/v3/getting-started/new-project.md:21]:
```suggestion
If you do not have a Android device, you can setup an emulator through Android Studio. You can see how to do that here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```

**Review by markhougaard — APPROVED** [review]

**Comment by markhougaard** [comment]
@timMikkelsen Very nice! LGTM 👍

**Pull request merged**